src/styles/blocks/catalog.scss

```
@@ -0,0 +1,37 @@
+@import "./../utils/mixin";
+@import "./../utils/variables";
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: 48px;
+  max-width: 944px;
+  margin: 0 auto;
+  
+  @media (min-width: $medium-size-media) and (max-width: $max-size-media) {
+    .catalog {
+      grid-template-columns: repeat(3, $card-width);
+    }
+  }
+  
+  @media (min-width: $min-size-media) and (max-width: $medium-size-media) {
+    .catalog {
+      grid-template-columns: repeat(2, $card-width);
+    }
+  }
+  
+  @media (max-width: $min-size-media) {
+    .catalog {
+      grid-template-columns: repeat(1, $card-width);
+    }
```

You can simplify it. Check if it'll work
```suggestion
   @media (min-width: $min-size-media) {
    .catalog {
      grid-template-columns: repeat(2, $card-width);
    }
  }
  
  @media (min-width: $medium-size-media) {
    .catalog {
      grid-template-columns: repeat(3, $card-width);
    }
  }
  
  @media (min-width:  $max-size-media) {
    .catalog {
      grid-template-columns: repeat(4, $card-width);
    }
```